src/Bridges/scalarizebridge.jl

```diff
@@ -0,0 +1,93 @@
+const VectorLPSet = Union{MOI.Zeros, MOI.Nonnegatives, MOI.Nonpositives}
+
+vector_set(::T, ::MOI.Zeros) where T = MOI.EqualTo(zero(T))
+vector_set(::T, ::MOI.Nonpositives) where T = MOI.LessThan(zero(T))
+vector_set(::T, ::MOI.Nonnegatives) where T = MOI.GreaterThan(zero(T))
```

Comment:
Don't these make more sense being `scalar_set`, `scalar_set_type`, and `scalar_function_type`?

Reply:
+1 for scalar set